website_outline.html

```diff
@@ -0,0 +1,69 @@
+<html>
+
+    <head>
+        <title>hannah kern</title>
+    </head>
+
+    <h1>
+        about
+    </h1>
+
+    <h1>
+        projects
+    </h1>
+
+    <h1>
+        contact
+    </h1>
+
+    <h1>
+        hello.
+    </h1>
+
+    <h2>
+        my name is hannah. i am passionate about helping people through innovative design. 
+    </h2>
+
+    <p>
+        I’m Hannah, a human centered design and development student at Penn State University. I am passionate about 
+    </p>
+
+    <h3>
+        UXPA
+    </h3>
+
+    <p>
+        info about UXPA
+    </p>
+
+    <h3>
+        Zoom-bot
+    </h3>
+
+    <p>
+        The Zoom-bot is an interactive navigation and maintence interface that could be implemented on subways and trains all over the world to improve riders experience.
+    </p>
+
+    <h3>
+        Clinique Virtual Reality
+    </h3>
+
+    <p>
+        info about clinique
+    </p>
+
+    <h4>
+        Let's Chat!
+    </h4>
+
+    <h5>
+        resume
+    </h5>
+
+    <p>
+        Github icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a>
+        Gmail icons made by <a href="https://www.flaticon.com/authors/pixel-perfect" title="Pixel perfect">Pixel perfect</a> from <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a>
+        LinkedIn icons made by <a href="https://www.flaticon.com/authors/pixel-perfect" title="Pixel perfect">Pixel perfect</a> from <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a>
```

Comment:
They usually specify exactly what text you must include to use an icon - is this what they said? Do you need the creative commons lisence?

Reply:
@hvk5388 You never addressed this. Please at least respond to the comment...